src/components/bp-qr-code-component/bp-qr-code-component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, Input } from '@angular/core';
+import { CurrencyProvider } from '../../providers/currency/currency';
+
+@Component({
+  selector: 'bp-qr-code-component',
+  templateUrl: 'bp-qr-code-component.html'
+})
+export class BpQrCodeComponent {
```

Comment:
Since we're installing a plugin from npm, it shouldn't be necessary. The plugin should have all the component elements to work. Try to isolate these functions into the plugin.

Reply:
Maybe more than a component I should add this to the includes folder.

Reply:
My main problem is that for some reason I cannot use a ts variable for module-color, position-ring-color, position-center-color properties